appendices/appendix_0.md

```diff
@@ -0,0 +1,9 @@
+\appendix
+\clearpage 
+\appendixpage
+\addappheadtotoc
```

Comment:
Unfortunately I couldn't figure out how to get the appendices to nest together under "Appendix" in the bookmark tree.  Instead they print like chapters, but at least there's a "Appendix" marker separating the chapters from appendices.  We might be able to get latex to force these to nest with each other if we dig deep enough.